src/researches/buildKeywordForms.js

```diff
@@ -7,6 +7,36 @@ const includes = require( "lodash/includes" );
 const filter = require( "lodash/filter" );
 const isUndefined = require( "lodash/isUndefined" );
 const escapeRegExp = require( "lodash/escapeRegExp" );
+const unique = require( "lodash/uniq" );
+const flatten = require( "lodash/flatten" );
+
+/**
+ * Checks if the input word contains a normalized or a non-normalized apostrophe.
+ * If so generates a complementary form (e.g., "il'y a" > "il`a")
```

Comment:
You're using a backtick (\`) in your example, but you're transforming `'` and `’`.

Reply:
I added all 5 forms (including the normalized one) to the generator.